.github/workflows/tag.yml

```diff
@@ -0,0 +1,28 @@
+name: flask-rebar Pull Request Tests
+
+on:
+  release:
+    types:
+    - created
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install pep517
+      run: |
+        python -m pip install pep517 --user
+    - name: Build a binary wheel and a source tarball
+      run: |
+        python -m pep517.build .
+    - name: Publish distribution 📦 to PyPI
+      if: startsWith(github.event.ref, 'refs/tags')
```

Comment:
I am curious to know why this if is necessary

Reply:
I am not sure what it is used for right this second, it is just what is recommended by pypa https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/

Reply:
Ho I see, in the example they do it on `push` so they only release to Pypi if the commit is tag. We don't need this since we do it on release created :)

Reply:
Ahh yes, so we shouldn't need it.  I do remember that from earlier today, but i got distracted and forgot to remove this line.